deploy/install.sh

```diff
@@ -224,6 +225,15 @@ then
   oc apply -f https://raw.githubusercontent.com/kabanero-io/kabanero-operator/${RELEASE}/deploy/optional.yaml --selector=kabanero.io/component=kappnav
 fi
 
+# Install Keycloak if selected
+if [ "$ENABLE_KEYCLOAK" == "yes" ]
+then
+   #final official URL but is not tested yet
+   # bash <(curl -s https://raw.githubusercontent.com/kabanero-io/kabanero-operator/${RELEASE}/deploy/keycloak-install.sh)
+   # For testing purpose: temporarily using GSA server before final URL available
+    bash <(curl -s -u meichang:mypwd https://rtpgsa.ibm.com/projects/w/wassvt/PersonaTest/meichang/keycloak-install.sh)
```

Comment:
This URL can't be used as no one outside of IBM will be able to access it.  You need to host this on a public branch in github, or somewhere else.

Reply:
fixed